extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -158,18 +157,16 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
 
   /**
    * Updates template based on shopping data.
-   * @param {!Array<!ShoppingConfigDataDef>} shoppingData
+   * @param {!Object<!ShoppingConfigDataDef>} shoppingData
    * @private
    */
   updateTemplate_(shoppingData) {
-    const shoppingDataForPage = this.shoppingTags_.map(
-      (shoppingTag) => shoppingData[shoppingTag.getAttribute('data-product-id')]
-    );
+    const shoppingDataPerPage = Object.values(shoppingData[this.pageEl_.id]);
```

Comment:
Nit: there's nothing at all wrong with the way this line is written, but I think we ca help people who are unfamiliar with the JSON structure:
```
const productOnPageToConfig = shoppingData[this.pageEl_.id];
const productConfigsForPage = Object.values(shoppingDataForPage);
```
(Correct me if my variable names here are inaccurate)

Reply:
Good point, I added in something more like this:
```
    const productOnPageToConfig = shoppingData[this.pageEl_.id];
    const shoppingDataPerPage = Object.values(productOnPageToConfig);
```